bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/Tally/AccumulationResult.cs

```diff
@@ -0,0 +1,90 @@
+using ElectionGuard.Encryption.Ballot;
+
+namespace ElectionGuard.Decryption.Tally;
+
+public class AccumulationResult
```

Comment:
i need to annotate this class but this is what you get back when you pass in ballots to accumulate